aleph2_data_model/src/com/ikanow/aleph2/data_model/utils/ContextUtils.java

```diff
@@ -15,12 +15,32 @@
  ******************************************************************************/
 package com.ikanow.aleph2.data_model.utils;
 
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
 import org.checkerframework.checker.nullness.qual.NonNull;
 
+import com.google.inject.AbstractModule;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.ikanow.aleph2.data_model.interfaces.data_access.IAccessContext;
 import com.ikanow.aleph2.data_model.interfaces.data_analytics.IAnalyticsContext;
 import com.ikanow.aleph2.data_model.interfaces.data_import.IHarvestContext;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
 
-public class ContextUtils {
+public class ContextUtils extends AbstractModule  {
```

Comment:
hmm not sure about this.... XxxUtils classes should be containers of static functions, seems like this has become a little bit more...